geometry_msgs/Pose objectPose
float64 hoverDistance
float64 pressAmount
float64 rate
---
int64 success_bool
